docs/Modders/schemas/nations.json

```diff
@@ -39,6 +39,8 @@
           "defeated": { "type": "string" },
           "startIntroPart1": { "type": "string" },
           "startIntroPart2": { "type": "string" },
+          "goldenAgeName": { "type": "string" },
+          "enteredGoldenAge": { "type": "string" },
```

Comment:
We're against adding extra fields when things can be one via a Unique, so the correct answer here is to add a unique e.g. "Golden Age is called [comment]"